aries-site/src/examples/templates/list-views/ListScreenExample.js

```diff
@@ -23,6 +24,7 @@ import {
   System,
   Hpe,
 } from 'grommet-icons';
+import { ThemeContext } from 'styled-components';
```

Comment:
Why using styled-components context instead of Grommet's?

Reply:
Thanks, I've updated to use grommet themecontext and confirmed functionality persists.